gilded_rose.rb

```diff
@@ -1,48 +1,19 @@
+require 'cheese'
+require 'conjured_item'
+require 'item_mapper'
+require 'standard_item'
+require 'sulfuras'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.